### ISSUE TITLE
AUTHORS file

### ISSUE THREAD

**Issue by jonschlinkert** [update/updater-contributors#2]
per this issue https://github.com/update/update/issues/9, from @jamen. 

We can add a task that generates the AUTHORS file using the array of authors returned from the GitHub API, and/or directly from package.json authors.

**Comment by jonschlinkert** [comment]
fwiw, when I created this my thinking was that the main goal of the updater should be to update an object (like `authors` in package.json) with the latest list of contributors. Then we should be able to do just about anything else with that data, using templates or whatever.

**Comment by doowb** [comment]
We might be able to split the formatters out of `github-contributors` to be able to format any array of "people". Then an updater can get the authors object from package.json and format it into the authors file.

**Comment by jonschlinkert** [comment]
good idea, feel free to abstract that code out and create `format-people` or whatever. then just swap it out in `github-contributors`
